hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/supervisor/process/ConsumerProcess.java

```diff
@@ -199,23 +204,23 @@ public boolean equals(Object o) {
         if (this == o) return true;
         if (o == null || getClass() != o.getClass()) return false;
         ConsumerProcess that = (ConsumerProcess) o;
-        return Objects.equals(subscriptionName, that.subscriptionName);
+        return Objects.equals(getSubscriptionName(), that.getSubscriptionName());
     }
 
     @Override
     public int hashCode() {
-        return Objects.hash(subscriptionName);
+        return Objects.hash(getSubscriptionName());
     }
 
-    public SubscriptionName getSubscriptionName() {
-        return subscriptionName;
-    }
-
-    public Consumer getConsumer() {
-        return consumer;
+    public Subscription getSubscription() {
```

Comment:
Is `process.getSubscription().getQualifiedName()` the only one usage of this getter? If so, we could just expose `getSubscriptionName()` created below

Reply:
Unfortunatelly not :) there is one more usage which requires the Subscription object itself.